collectd/conf.go

```diff
@@ -144,6 +144,15 @@ LoadPlugin swap
 	"process":    ``,
 }
 
+func reservedIdPrefixError(
```

Comment:
Fyi, Go style would be more like:

    // reservedIdPrefixError returns an error message for when the user
    // tries to define a reserved ID.  The component should be "receiver",
    // "processor", or "exporter".
    func reservedIDPrefixError(component, id string) error {
      ...
    }

Reply:
Good to know. Changed.

Reply:
Also, put all the parameters on one line.